project5/3700kvstore

```diff
@@ -57,7 +58,56 @@ class SimulatorSocket:
         self.sock.send(msg)
 
 
-class Server:
+class LoggingMixin:
```

Comment:
I would suggest adding a purpose statement for each of these functions, but if you don't want to, that's okay lol.